app/views/sessions/_form.html.erb

```diff
@@ -1,24 +1,41 @@
+<% content_for :head do %>
+  <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
+  <%= javascript_pack_tag 'input' %>
+<%end%>
+
 <%= form_for :session, url: session_path do |form| %>
-  <div class="text-field">
-    <%= form.label :email %>
-    <%= form.text_field :email, type: 'email', class: "input-field" %>
-  </div>
+  <div class="flex flex-col items-center px-4 py-2 md:px-32 lg:px-60 py-10 lg:py-14">
+    <div class="content-center text-center py-4 md:w-max mt-2.5"> 
+      <%= render "shared/header_stamp" %>
+      <h2 class="header-2 w-80 md:w-11/12 md:ml-12">Para continuar inicia sesión en tribo</h2>
+    </div>  
+      
+      <div class="text-field mt-8 md:w-full">
+        <%= form.text_field :email, type: 'email', class: "input-field w-60 md:w-full", placeholder: "Correo electrónico", required: true  %>
+      </div>
 
-  <div class="password-field">
-    <%= form.label :password %>
-    <%= form.password_field :password, class: "input-field" %>
-  </div>
+      <div class="password-field mt-4 md:w-full">
+        <%= form.password_field :password, class: "input-field w-60 md:w-full", placeholder: "Contraseña", required: true  %>
+      </div>
 
-  <div class="submit-field">
-    <%= form.submit %>
-  </div>
+      <div class="submit-field mt-8 md:w-full">
+        <%= form.submit "Iniciar sesión", class: 'button btn-prim justify-center w-60 md:w-full' %>
+      </div>
+
+
+      <hr class="w-60 mt-8 md:w-full">
 
-  <div class="other-links">
-    <% if Clearance.configuration.allow_sign_up? %>
-      <%= link_to t(".sign_up"), sign_up_path %>
-    <% end %>
-    <!--
-    <%= link_to t(".forgot_password"), new_password_path %>
-    -->
+      <div class="other-links mt-4">
+        <% if Clearance.configuration.allow_sign_up? %>
```

Comment:
It'd be nice having a helper method like `allow_sign_up` without needing to make full reference to the `Clearance` class.